### DISCUSSION TITLE
GitHub Student account activation

### DISCUSSION THREAD

**Discussion by muhammadali03-ai** [community/community#64001] [GitHub Education]
Hello everyone, my application for student package was not verified because of the following things: 

**You are unlikely to be verified until you have completed your GitHub user profile with your full name exactly as it appears in your academic affiliation document plus a short bio. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.**

The image you selected does not appear to contain your school name. Ensure your image (photo or upload) is legible. We do not accept school logos. If you are taking a photo of a screen try to have it as large a resolution as possible. You may include multiple documents in your image. If your original document is not in English then you might take a photo of the original next to an English translation.

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.**
 The reason for the problem with location is that now I'm on an academic Break and thus living in my country for some time. So the location i sent is not the location which is close to the campus. My university campus is in south Korea 
The reason for the photo is that the camera could not capture it well. I tried both laptop and mobile phone cameras. Some of the text on my student id is yellow color especially the date of enrollment which is very hard to read.

Hope you have a solution. 
Thank you:)

**Comment by tooltham** [comment]
You have a problem with your identity verification documents. The letters on your student ID card could be easier to read because it is yellow. Another option is to use a student recommendation letter issued by your institution. I recommend this method to my students. Because the document is more precise and has higher credibility, it only took a few days for my students to verify themselves.

**Reply by muhammadali03-ai** [reply]
Alright. Thank you for your help.)
